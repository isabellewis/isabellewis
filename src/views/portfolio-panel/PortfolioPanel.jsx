import React from 'react';
import styled from 'styled-components';

import { PortfolioBox } from '../../components';

const PortfolioWrapper = styled.div`
  min-height: 100%;
  padding: 30px 0;
`;

const projectData = {
  projects: [
    {
      name: '0 First react project',
      technologies: 'HTML, CSS, JavaScript, Webpack',
      github: 'https://github.com/isabellewis/isabellewis',
      livewebsite: 'https://github.com/isabellewis/isabellewis',
      description: 'This project is all about how to do this in react'
    },
    {
      name: '1 Second project',
      technologies: 'HTML, CSS, JavaScript',
      github: 'https://github.com/isabellewis/isabellewis',
      livewebsite: 'https://github.com/isabellewis/isabellewis',
      description: 'This project is using web fundamentals to build something cool'
    },
    {
      name: '2 Third project',
      technologies: 'HTML, CSS, JavaScript, Webpack',
      github: 'https://github.com/isabellewis/isabellewis',
      livewebsite: 'https://github.com/isabellewis/isabellewis',
      description: 'Descriptions will go here'
    },
    {
      name: '3 Fourth project here',
      technologies: 'HTML, CSS, JavaScript, Webpack',
      github: 'https://github.com/isabellewis/isabellewis',
      livewebsite: 'https://github.com/isabellewis/isabellewis',
      description: 'Descriptions will go here'
    },
    {
      name: '4 The fifth and final project',
      technologies: 'HTML, CSS, JavaScript, Webpack',
      github: 'https://github.com/isabellewis/isabellewis',
      livewebsite: 'https://github.com/isabellewis/isabellewis',
      description: 'Descriptions will go here'
    }
  ]
}

export const PortfolioPanel = () => (
  <PortfolioWrapper id="portfolioPanel">
    <PortfolioBox projectData={projectData} />
  </PortfolioWrapper>
);
