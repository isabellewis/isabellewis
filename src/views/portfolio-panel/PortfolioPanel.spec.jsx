import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import { PortfolioPanel } from './PortfolioPanel';

describe('views/PortfolioPanel', () => {
	let component;

	beforeEach(() => {
		component = shallow(<PortfolioPanel />);
	});

	it('should match the snapshot', () => {
		expect(component).toMatchSnapshot();
	});
});