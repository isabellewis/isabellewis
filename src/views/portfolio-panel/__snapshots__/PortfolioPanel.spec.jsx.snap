// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`views/PortfolioPanel should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <PortfolioPanel />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": <PortfolioBox
        projectData={
          Object {
            "projects": Array [
              Object {
                "description": "This project is all about how to do this in react",
                "github": "https://github.com/isabellewis/isabellewis",
                "livewebsite": "https://github.com/isabellewis/isabellewis",
                "name": "0 First react project",
                "technologies": "HTML, CSS, JavaScript, Webpack",
              },
              Object {
                "description": "This project is using web fundamentals to build something cool",
                "github": "https://github.com/isabellewis/isabellewis",
                "livewebsite": "https://github.com/isabellewis/isabellewis",
                "name": "1 Second project",
                "technologies": "HTML, CSS, JavaScript",
              },
              Object {
                "description": "Descriptions will go here",
                "github": "https://github.com/isabellewis/isabellewis",
                "livewebsite": "https://github.com/isabellewis/isabellewis",
                "name": "2 Third project",
                "technologies": "HTML, CSS, JavaScript, Webpack",
              },
              Object {
                "description": "Descriptions will go here",
                "github": "https://github.com/isabellewis/isabellewis",
                "livewebsite": "https://github.com/isabellewis/isabellewis",
                "name": "3 Fourth project here",
                "technologies": "HTML, CSS, JavaScript, Webpack",
              },
              Object {
                "description": "Descriptions will go here",
                "github": "https://github.com/isabellewis/isabellewis",
                "livewebsite": "https://github.com/isabellewis/isabellewis",
                "name": "4 The fifth and final project",
                "technologies": "HTML, CSS, JavaScript, Webpack",
              },
            ],
          }
        }
      />,
      "id": "portfolioPanel",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "projectData": Object {
          "projects": Array [
            Object {
              "description": "This project is all about how to do this in react",
              "github": "https://github.com/isabellewis/isabellewis",
              "livewebsite": "https://github.com/isabellewis/isabellewis",
              "name": "0 First react project",
              "technologies": "HTML, CSS, JavaScript, Webpack",
            },
            Object {
              "description": "This project is using web fundamentals to build something cool",
              "github": "https://github.com/isabellewis/isabellewis",
              "livewebsite": "https://github.com/isabellewis/isabellewis",
              "name": "1 Second project",
              "technologies": "HTML, CSS, JavaScript",
            },
            Object {
              "description": "Descriptions will go here",
              "github": "https://github.com/isabellewis/isabellewis",
              "livewebsite": "https://github.com/isabellewis/isabellewis",
              "name": "2 Third project",
              "technologies": "HTML, CSS, JavaScript, Webpack",
            },
            Object {
              "description": "Descriptions will go here",
              "github": "https://github.com/isabellewis/isabellewis",
              "livewebsite": "https://github.com/isabellewis/isabellewis",
              "name": "3 Fourth project here",
              "technologies": "HTML, CSS, JavaScript, Webpack",
            },
            Object {
              "description": "Descriptions will go here",
              "github": "https://github.com/isabellewis/isabellewis",
              "livewebsite": "https://github.com/isabellewis/isabellewis",
              "name": "4 The fifth and final project",
              "technologies": "HTML, CSS, JavaScript, Webpack",
            },
          ],
        },
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <PortfolioBox
          projectData={
            Object {
              "projects": Array [
                Object {
                  "description": "This project is all about how to do this in react",
                  "github": "https://github.com/isabellewis/isabellewis",
                  "livewebsite": "https://github.com/isabellewis/isabellewis",
                  "name": "0 First react project",
                  "technologies": "HTML, CSS, JavaScript, Webpack",
                },
                Object {
                  "description": "This project is using web fundamentals to build something cool",
                  "github": "https://github.com/isabellewis/isabellewis",
                  "livewebsite": "https://github.com/isabellewis/isabellewis",
                  "name": "1 Second project",
                  "technologies": "HTML, CSS, JavaScript",
                },
                Object {
                  "description": "Descriptions will go here",
                  "github": "https://github.com/isabellewis/isabellewis",
                  "livewebsite": "https://github.com/isabellewis/isabellewis",
                  "name": "2 Third project",
                  "technologies": "HTML, CSS, JavaScript, Webpack",
                },
                Object {
                  "description": "Descriptions will go here",
                  "github": "https://github.com/isabellewis/isabellewis",
                  "livewebsite": "https://github.com/isabellewis/isabellewis",
                  "name": "3 Fourth project here",
                  "technologies": "HTML, CSS, JavaScript, Webpack",
                },
                Object {
                  "description": "Descriptions will go here",
                  "github": "https://github.com/isabellewis/isabellewis",
                  "livewebsite": "https://github.com/isabellewis/isabellewis",
                  "name": "4 The fifth and final project",
                  "technologies": "HTML, CSS, JavaScript, Webpack",
                },
              ],
            }
          }
        />,
        "id": "portfolioPanel",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "projectData": Object {
            "projects": Array [
              Object {
                "description": "This project is all about how to do this in react",
                "github": "https://github.com/isabellewis/isabellewis",
                "livewebsite": "https://github.com/isabellewis/isabellewis",
                "name": "0 First react project",
                "technologies": "HTML, CSS, JavaScript, Webpack",
              },
              Object {
                "description": "This project is using web fundamentals to build something cool",
                "github": "https://github.com/isabellewis/isabellewis",
                "livewebsite": "https://github.com/isabellewis/isabellewis",
                "name": "1 Second project",
                "technologies": "HTML, CSS, JavaScript",
              },
              Object {
                "description": "Descriptions will go here",
                "github": "https://github.com/isabellewis/isabellewis",
                "livewebsite": "https://github.com/isabellewis/isabellewis",
                "name": "2 Third project",
                "technologies": "HTML, CSS, JavaScript, Webpack",
              },
              Object {
                "description": "Descriptions will go here",
                "github": "https://github.com/isabellewis/isabellewis",
                "livewebsite": "https://github.com/isabellewis/isabellewis",
                "name": "3 Fourth project here",
                "technologies": "HTML, CSS, JavaScript, Webpack",
              },
              Object {
                "description": "Descriptions will go here",
                "github": "https://github.com/isabellewis/isabellewis",
                "livewebsite": "https://github.com/isabellewis/isabellewis",
                "name": "4 The fifth and final project",
                "technologies": "HTML, CSS, JavaScript, Webpack",
              },
            ],
          },
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
